src/ui/buttons/buttonDesigns/alternateSlideLogoDesign.jsx

```diff
@@ -0,0 +1,127 @@
+/* @flow */
+/** @jsx node */
+import { LOGO_CLASS } from '@paypal/sdk-logos/src';
+import { node, Fragment, type ChildType } from 'jsx-pragmatic/src';
+
+import { CLASS } from '../../../constants';
+import { BUTTON_SIZE_STYLE } from '../config';
+
+import type {  ContentOptions } from './types';
+
+type AnimationProps = {|
+    paypalLabelContainerElement : Object,
+    paypalLogoStartingLeftPosition : string
+|};
+
+
+export const FADEOUT_LOGO_SHOW_LABEL_TEXT_CONFIG = {
+    min:                            BUTTON_SIZE_STYLE.tiny.minWidth,
+    max:                            BUTTON_SIZE_STYLE.medium.maxWidth,
+    cssClasses: {
+        DOM_READY:                  CLASS.DOM_READY,
+        ANIMATION_CONTAINER:        'personalized-design-container',
```

Comment:
This can be removed and replaced with a hard coded value since it is the same everywhere